README.md

```diff
@@ -132,6 +132,8 @@ Data which is stored upon startup includes:
 
 * Now, you can press the Start button in Visual Studio to run the server (it is a green play button in the top middle of the tool bar). It will open the web browser and, after a period that should just be a few minutes long but sometimes lasts half an hour or more, display an Error 403.14 - Forbidden. This is expected. You can now begin manually testing the API.
 
+* If you haven't already, do an `npm install`. This only needs to be done once, then you can just `npm start`
```

Comment:
Since the steps just before this were in the gordon-360-api repo, it would be helpful to say clearly that these npm commands should be run in the gordon-360-ui repo.